modules/tensor_mechanics/test/tests/dynamics/wave_1D/tests

```diff
@@ -34,6 +34,31 @@
                    " in a linear elastic material with both Rayleigh damping and"
                    " numerical damping resulting from Hilber-Hughes-Taylor (HHT) time integration."
   [../]
+  [./rayleigh_hht_ad]
+    type = 'Exodiff'
+    input = 'wave_rayleigh_hht_AD.i'
+    exodiff = 'wave_rayleigh_hht_out.e'
+    abs_zero = 1e-09
+    compiler = 'GCC CLANG'
```

Comment:
do we still need this restriction? We dropped intel compiler support a few months ago.